app/views/govuk_publishing_components/components/docs/success_alert.yml

```diff
@@ -0,0 +1,16 @@
+name: Success Alert
+description: Used at the top of the page, to summarise a successful user action.
+accessibility_criteria: |
+  - should be focused on page load, to ensure this error is noticed by assistive tech
```

Comment:
This isn't right, it's not an error. We should probably still include `tabindex=-1` so that the alert is focusable.

Reply:
@fofr doh! missed that when copying.

I think I have three options here.

1) give it a tabindex of -1 and set up a module as per https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/javascripts/govuk_publishing_components/components/error-summary.js so it's auto focused on page load
2) give it a tabindex of 0 so it is tabbable (which would be inconsistent with error-summary) 
3) Not give it one, treat it as ordinary flow

I'm tempted to go down the route of 3 as that seems the path of least risk of controversy. What do you think? I'll set this up now.

Reply:
Option 1, but without the JS:
https://github.com/alphagov/govuk_publishing_components/pull/254/files#r178077033

Docs should also indicate the autofocus behaviour.